ironfish/src/fileStores/config.ts

```diff
@@ -388,6 +397,8 @@ export class Config extends KeyStore<ConfigOptions> {
       rpcTcpPort: 8020,
       tlsKeyPath: files.resolve(files.join(dataDir, 'certs', 'node-key.pem')),
       tlsCertPath: files.resolve(files.join(dataDir, 'certs', 'node-cert.pem')),
+      rpcHttpHost: 'localhost',
```

Comment:
This doesn't work for ipv6

Reply:
won't this resolve just fine to `127.0.0.1` on an ipv4 system and `::1` on an ipv6 system? I don't quite follow @NullSoldier

Reply:
No we ran into an issue where if you listen on localhost, ipv6 connections won't work because of host node sockets work. We had to switch the pool host over. See this, https://github.com/iron-fish/ironfish/pull/3356/files